src/chapters/forms/studio.rst

```diff
@@ -0,0 +1,156 @@
+Studio: HTTP and Forms
+======================
+
+Introduction
+------------
+
+This chapter taught you that forms submit data in HTTP requests. This studio uses form and HTTP concepts to build a *search engine selector*, that is, a search form that allows a user to choose which search engine they would like to use. It will look like this:
+
+.. figure:: figures/search-engine-selector.png
+   :alt: A form with a text input and radio buttons corresponding to various search engines.
+
+   The search engine selector that we will build.
+
+Most search engines work the same way. The have a single text input, and they submit data using a ``GET`` request. Additionally, many of the most popular search engines also use the same name for the search parameter, ``q``. 
+
+.. admonition:: Try It!
+
+   Use 2-3 different search engines to search for the same term. On the results page, look at the URL. Did the search happen via ``GET`` or ``POST``? If a ``GET`` request was made, what is the name of the parameter containing your search term?
+
+   *Note:* You may have to copy/paste the URL into a text editor to find the search parameter. Some engines add other parameters to the URL, causing it to extend past the end of the browser's address bar.
+
+.. note:: We remarked previously that most forms use ``POST`` because they cause data to be changed on the server. A web search only *retrieves* data. It does not change data. Therefore it's safe to use a ``GET`` request for searches.
+
+The fact that most search engines use the name ``q`` for their search boxes will allow us to easily create a form that is capable of sending a search request to several search engines.
+
+The form will send a request with query parameter ``q`` to the selected engine. Since this request looks essentially the same as requests coming from the search engine's own form (for examle, at `google.com <https://google.com>`_) it will give us back the results the same as if we had searched via those sites. 
+
+Getting Started
+---------------
+
+Fork the `starter code for this studio <https://repl.it/@launchcode/Forms-Studio>`_, which has some basic HTML to get you started. 
```

Comment:
Should we change this to *git* instructions?

Reply:
Yeah, we should. I'm not it.

Reply:
I added instructions for them to do this locally, but nothing about Git. We don't have a standardized way of setting this up, which feels like something we should figure out collectively.